### ISSUE TITLE
[BUG] Dask-cudf Read Parquet Performance Regression

### ISSUE THREAD

**Issue by VibhuJawa** [rapidsai/cudf#3695]
**Describe the bug**

There seems to be a 2x+ performance regression for reading parquets with dask-cudf.  The regression seems to occur b/w 2nd December's nightly and 3rd December's nightly. 

Performance of below code:

1.  6 Jan (Latest nightly) :  `47.822 s`
2. 2nd December Nightly: `19.737252712 s`
3. 3rd December Nightly: `46.665285587 s`

**Steps/Code to reproduce bug**

```python
from dask_cuda import LocalCUDACluster
from dask.distributed import Client
import dask_cudf
import time

### see slack for directory details
data_refresh_dir = xxx

if __name__ == "__main__":
    cluster = LocalCUDACluster()
    client = Client(cluster)
    
    table_name = 'web_clickstreams'
    web_clickstreams_cols = ['wcs_user_sk','wcs_click_date_sk','wcs_item_sk','wcs_sales_sk']
    web_clickstreams_df = dask_cudf.read_parquet(data_refresh_dir + f'/{table_name}/*.parquet', columns = web_clickstreams_cols)
   
    st = time.time()
    len_df = len(web_clickstreams_df)
    et = time.time()
    
    print("Time taken = {}s".format(et-st))
    print("Len of DF = {:,}".format(len_df))

    client.close()
    cluster.close()
```

**Environment overview (please complete the following information)**
 - Environment location:  conda
 - Method of cuDF install: [conda]

#### 2nd Dec env instructions:
```
conda create --name  cudf_11_2_dec

conda install -c rapidsai-nightly -c nvidia -c conda-forge -c defaults cudf=0.11.0a191202 dask-cudf=0.11.0a191202 dask-cuda=0.11.0b191202 dask=2.9.0
```

#### 3rd Dec env instructions:
```
conda create --name  cudf_11_3_dec

conda install -c rapidsai-nightly -c nvidia -c conda-forge -c defaults cudf=0.11.0a191203 dask-cudf=0.11.0a191203 dask-cuda=0.11.0b191203 dask=2.9.0
```

**Environment details**
2nd December Nightly
```
$ conda list | grep cudf
# packages in environment at /raid/vjawa/conda_install/conda_env/envs/cudf_11_2_dec:
cudf                      0.11.0a191202         py37_3726    rapidsai-nightly
dask-cudf                 0.11.0a191202         py37_3726    rapidsai-nightly
libcudf                   0.11.0b191211      cuda10.1_105    rapidsai-nightly
$ conda list | grep dask
dask                      2.9.0                      py_0    conda-forge
dask-core                 2.9.0                      py_0    conda-forge
dask-cuda                 0.11.0b191202            py37_9    rapidsai-nightly
```

3rd December Nightly
```

$ conda list | grep dask
dask                      2.9.0                      py_0  
dask-core                 2.9.0                      py_0  
dask-cuda                 0.11.0b191203           py37_13    rapidsai-nightly
dask-cudf                 0.11.0a191203         py37_3788    rapidsai-nightly
$ conda list | grep cudf
# packages in environment at /raid/vjawa/conda_install/conda_env/envs/cudf_11_nightly_3_dec:
cudf                      0.11.0a191203         py37_3788    rapidsai-nightly
dask-cudf                 0.11.0a191203         py37_3788    rapidsai-nightly
libcudf                   0.11.0a191203     cuda10.1_3788    rapidsai-nightly
```

**Additional context**

The performance of `cudf` remains almost the same so i expect the performance drop at `dask` level. 

For data files please see internal slack.

**Comment by rjzamora** [comment]
Sorry about this @VibhuJawa - I submitted [#3520](https://github.com/rapidsai/cudf/pull/3520/) right after causing the "regression" (really just a poor choice of default arguments) in #3407

**Comment by VibhuJawa** [comment]
> Sorry about this @VibhuJawa - I submitted [#3520](https://github.com/rapidsai/cudf/pull/3520/) right after causing the "regression" (really just a poor choice of default arguments) in #3407 -- The fix seems to have gotten pushed back by release-cycle churn

 Thanks for looking into this. No worries :-)

**Comment by kkraus14** [comment]
Solved by #3520
